connect/src/com/telenor/connect/ConnectSdk.java

```diff
@@ -359,19 +428,13 @@ public void onFailure(Call<WellKnownAPI.WellKnownConfig> call, Throwable error)
                     }
                 });
 
-        connectivityManager
-                = (ConnectivityManager) context.getSystemService(Context.CONNECTIVITY_SERVICE);
-        if (connectivityManager != null && Build.VERSION.SDK_INT >= Build.VERSION_CODES.LOLLIPOP) {
-            initalizeCellularNetwork();
-            initalizeDefaultNetwork();
-        }
+        HeLogic.initializeNetworks(context, useStaging);
         initializeAdvertisingId(context);
         isInitialized = true;
-        tsSdkInitiliazation = System.currentTimeMillis();
+        tsSdkInitialization = System.currentTimeMillis();
     }
 
-
-    public static String getMccMnc() {
+    private static String getMccMnc() {
```

Comment:
Two abbreviations in one method, well done ;D 
Not sure if it's neccessary, but consider spelling out one or two of the abbreviations to hint at people like me what this is about.

Reply:
I guess many developers don't know all abbreviations from the telecom world, so then it makes sense to spell it out. I don't think this is an abbreviation used much in the Connect sphere, is it? In that case, perhaps spell it out. Unless it makes sense for us to learn that abbreviation, then keep it :)